src/components/Stepper/Stepper.test.tsx

```
@@ -0,0 +1,337 @@
+      const { container } = render(<Stepper steps={[]} />);
+      const stepsContainer = container.querySelector('[role="tablist"]');
+      expect(stepsContainer?.children).toHaveLength(0);
+    });
+
+    it('handles default parameter values being used', () => {
+      // This specifically tests the internal Step component with undefined parameters
+      // to trigger the default value branches
+
+      // Test by calling Stepper without passing stepClassName explicitly
+      const { container } = render(
+        // @ts-ignore - intentionally passing undefined to test default parameter branches
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)